modin/conftest.py

```diff
@@ -342,6 +348,133 @@ def make_csv_file():
     teardown_test_files(filenames)
 
 
+def create_and_call(filenames, filename, force, nrows, ncols, func: str, func_kw=None):
+    if force or not os.path.exists(filename):
+        df = pandas.DataFrame({f"col{x + 1}": np.arange(nrows) for x in range(ncols)})
+        getattr(df, func)(filename, **func_kw if func_kw else {})
+        filenames.append(filename)
+
+
+def make_default_file(file_type: str):
+    filenames = []
+    func = None
+
+    if file_type == "json":
+
+        def _make_json_file(filename, nrows=NROWS, ncols=2, lines=False, force=True):
+            func_kw = {"lines": lines, "orient": "records"} if lines else {}
+            create_and_call(
```

Comment:
Looks like test DataFrame will be created for each of the tests, can it be cached somehow?

Reply:
Creating a dataframe with default sizes takes no more than 0.02 seconds. I think there is no need yet.

Reply:
Ok, let's leave it as is.